packages/components/atoms/f-form-field/src/components/_tests/FormSelectionControl.test.js

```diff
@@ -51,32 +51,6 @@ describe('FormSelectionControl', () => {
     });
 
     describe('computed ::', () => {
-        describe('isCheckbox ::', () => {
```

Comment:
Super small one (can you tell I'm clearing out old emails today) – when you get 5 mins, could you make a EE ticket to add some tests in to test that the class using the `inputType` is populated as expected (so when radio it has a class of `c-formField-field--radio`).